src/components/Card/Card.css

```diff
@@ -11,9 +10,7 @@
 }
 
 .results--image {
-  height: 130px;
-  width: 130px;
-  margin-top: 40px;
```

Comment:
I personally think the size of the imgs here should be hardcoded, accounts for weird sizings and extra pixelazation.  Getting rid of the media queries for grid columns was a good call tho.

Reply:
I agree with the pixel issue. But I feel that hard-coding them made the space between them really wide. I think we should take a look and make a decision. If we hard code it, it should be in the grid sizing. I found 161px to be the best size and not being pixilated. 
Maybe we can find a way to limit the size to between 130-161px and they only adjust within those dims?

Reply:
Cool yeah, we can talk about it and revisit it in general styling updates.